daemon/api/dependencies.py

```diff
@@ -77,27 +116,77 @@ def localpath(self) -> Path:
                 detail=f'File `{self.filename}` not found in workspace `{self.workspace_id}`',
             )
 
-    @cached_property
-    def port_expose(self) -> str:
+    @property
+    def newfile(self) -> str:
+        """return newfile path in format
+        `<root-workspace>/<workspace-id>/<flow-id>_<original-filename>`
+
+        :return: return filepath to save flow config in
         """
-        Sets `port_expose` for the Flow started in `partial-daemon`.
-        This port needs to be exposed before starting `partial-daemon`, hence set here.
-        If env vars are passed, set them in current env, to make sure all values are replaced.
-        Before loading the flow yaml, change CWD to workspace dir.
+        return get_workspace_path(self.workspace_id, f'{self.id}_{self.filename}')
 
-        :return: port_expose
+    def load_and_dump(self) -> None:
```

Comment:
@maximilianwerk This is the discussion we had about `load` & `save` config for Flows before starting containers.